File: Data/Dirigent.cs
using System.ComponentModel.DataAnnotations;

namespace MaestroNotes.Data
{
    public class Dirigent
    {
        [Key]
        public int Id { get; set; }
        [MaxLength(50)]
        public string Vorname { get; set; } = "";
        [MaxLength(50)]
        public string Name { get; set; } = "";
        public DateTime? Born { get; set; }
        [MaxLength(1000)]
        public string? Note { get; set; }

        public DateTime? Died { get; set; }
        // ALTER TABLE Dirigenten ADD Died DATETIME NULL;
    }
}

File: php-app/app/Models/Dirigent.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Dirigent extends Model
{
    use HasFactory;

    protected $table = 'dirigenten';

    protected $fillable = [
        'vorname',
        'name',
        'born',
        'note',
    ];

    protected $casts = [
        'born' => 'date',
    ];
}
